tests/Functional/Controller/MenuTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Functional\Controller;
+
+use Sonata\AdminBundle\Tests\App\AppKernel;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+final class MenuTest extends WebTestCase
+{
+    public function testDynamicMenuInLongRunningProcess(): void
+    {
+        $client = static::createClient();
+        $client->disableReboot();
```

Comment:
This is what makes the client share the same container? If yes, can you add a comment here to explain this line (ex: this ensures the container is reused across multiple request to simulate the behavior of ...)?

Reply:
sure, adding